requirements.txt

```diff
@@ -6,4 +6,4 @@ prompt_toolkit==2.0.4
 SQLAlchemy==1.2.11
 six==1.11.0
 requests==2.20.0
-moz_sql_parser==2.18.18240
```

Comment:
This version causes a lot of trouble since it didn't pin `mo_future` and the latest version of `mo_future` is incompatible with this version

Reply:
see: https://github.com/mozilla/moz-sql-parser/issues/98